cmd/chaos-multiversion-helper/autoconvert/convert.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package autoconvert
+
+import (
+	"os"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/chaos-mesh/chaos-mesh/cmd/chaos-multiversion-helper/common"
+)
+
+func NewConvertCmd(log logr.Logger) *cobra.Command {
+	var version, hub string
+
+	var cmd = &cobra.Command{
+		Use:   "autoconvert --version <version> --hub <hub-version>",
+		Short: "Generates code to automatically convert between two versions",
+		Long: `Autoconvert will do the following things:
+		1. remove the Hub declaration in <version>, if it is.
+		2. create the Hub tag for <hub-version>, if it is not.
+		3. generate ConvertTo and ConvertFrom function for the <version>, and assume it has 
+			a type which is deeply the same with the <hub-version>.
+		`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return run(log, version, hub)
+		},
+	}
+
+	cmd.Flags().StringVar(&version, "version", "", "the version to generate the convert")
+	cmd.Flags().StringVar(&hub, "hub", "", "the hub version")
+
+	cmd.MarkFlagRequired("version")
+	cmd.MarkFlagRequired("hub")
+
+	return cmd
+}
+
+func removeHub(version string) error {
+	generatedHubFile := common.ChaosMeshAPIPrefix + version + "/" + "zz_generated.hub.chaosmesh.go"
+	err := os.Remove(generatedHubFile)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return nil
+		}
+
+		return errors.Wrapf(err, "remove file %s", generatedHubFile)
+	}
```

Comment:
I found that `autoconvert` failed to remove the hub file.

The value of `generatedHubFile` is mostly like `github.com/chaos-mesh/chaos-mesh/api/v1alpha1/zz_generated.hub.chaosmesh.go`, it is not a filepath could be removed by `os.Remove`